src/core/meshcop/border_agent.hpp

```diff
@@ -338,6 +350,7 @@ class BorderAgent : public InstanceLocator, private NonCopyable
     Ip6::Udp::Receiver         mUdpReceiver;
     Ip6::Netif::UnicastAddress mCommissionerAloc;
     TimeoutTimer               mTimer;
+    uint16_t                   mUdpPort;
```

Comment:
Suggest moving up after `State`? (we generally aim to keep member variables of similar/smaller size first).